jupyterlab-dataops/resources/cloudbeaver/conf/initial-data-sources-bigquery.conf.j2

```diff
@@ -11,10 +11,11 @@
 			"configuration": {
 				"host": "https://www.googleapis.com/bigquery/v2",
 				"port": "443",
+				"database": "{{ project_id }}",
 				"url": "jdbc:bigquery://https://www.googleapis.com/bigquery/v2:443",
 				"type": "dev",
 				"properties": {
-					"OAuthPvtKeyPath": "/home/jupyter/gcp-service-account-key.json"
+					"OAuthType": "3"
```

Comment:
What is this "3"?

Reply:
That's enum for authenticating with Google Authentication Default Credentials. I can provide link with docs for JDBC BigQuery driver in comment above.